README.md

```diff
@@ -101,6 +101,7 @@ Your\Namespace\SpecialPageWithRelatedDataObject:
     -
       RelatedDataObjects:
         type: nested
+        relationClass: App\DataObjects\Tags # Will be pulled from has_many / many_many, but you can specify it here too
```

Comment:
Where does `App\DataObjects\Tags` come from? Looks like an application namespace?

Reply:
It's just an example / sample relation class. Yeah it's user-code.